### PR TITLE
create modles animal commnet like

### PR DESCRIPTION
**Pull request by salsabeelomar**

### CODE REVIEW

File: src/models/animal.model.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    type: DataTypes.UUID,
+    autoIncrement: true
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false
+  }
+}, {
+  sequelize,
+  modelName: 'animal'
```
Comment (aaamra) [inline src/models/animal.model.ts:19]: i think this line is optional
sequalize creates the table with the model name in plural
so it's not preferred to override the names in sequalize

File: src/models/animal.model.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+})
+// export async function createLike () {
+//   await Animal.create({ name: 'Cat' })
+// }
```
Comment (aaamra) [inline src/models/animal.model.ts:23]: delete this commented code

File: src/models/comment.model.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    type: DataTypes.INTEGER
+  }
+}, {
+  sequelize,
+  modelName: 'comment'
+})
+
+// export async function createComment () {
+//   await Comment.create({ comment: 'first', postId: 1, userId: 1, image: 'https://img.freepik.com/premium-photo/cute-dog-during-consultation_23-2149314361.jpg' })
+// }
```
Comment (aaamra) [inline src/models/comment.model.ts:43]: same issues here

File: src/models/like.model.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    type: DataTypes.INTEGER
+  }
+}, {
+  sequelize,
+  modelName: 'like'
+})
+
+// export async function createLike () {
+//   await Like.create({ postId: 1, userId: 1 })
+// }
```
Comment (aaamra) [inline src/models/like.model.ts:27]: also same here

**Review by aaamra — CHANGES_REQUESTED** [review]

**Review by aaamra — COMMENTED** [review]
i liked your file naming convention in models
i think this convention exists in nest.js framework

but we are not using nest.js
so we need to change it to our convention
so instead of `animal.model.ts` => `Animal.ts`

**Review by Saeed99Madi — APPROVED** [review]

**Review by AseelL — APPROVED** [review]

**Review by mohmmed23 — APPROVED** [review]

**Pull request merged**